### ISSUE TITLE
Add "Cleanup" task to ensure that /var/run is a link to /run

### ISSUE THREAD

**Issue by ferricoxide** [plus3it/AMIgen7#53]
### Problem Description:

On instances launched RHEL 7 AMIs `/var/run` is not a link to `/run`. This breaks DBUS upgrades

### Expected Behavior:

`/var/run` is a link to `/run` in all created AMIs

### Actual Behavior:

`/var/run` is not a link to `/run` in all created AMIs

### (Detailed) Steps to reproduce:

Build new RHEL 7 AMIs from current scripts

### (Optional) Fix recommendation:

Add an "end-of-build" type of task to ensure that `/var/run` is a link to `/run`.

**Comment by ferricoxide** [comment]
@lorengordon you addressed this via [spel](/plus3it/spel)?

**Comment by lorengordon** [comment]
No. Seems more appropriate to _fix_ in this project. In spel was only planning to _test_ that the symlink existed on a built AMI.

**Comment by eemperor** [comment]
I have it in my queue to create that symlink test.

**Comment by ferricoxide** [comment]
I think that Red Hat fixed the issue that was provoking this not long after the problem was reported (though, may just be that, having moved on to more-recent bootstrap AMIs, we're simply no longer triggering the problem)

At any rate: @eemperor did you create the symlink test? If not, should we close this issue, regardless?
